registry.md

```diff
@@ -12,6 +12,9 @@ This page lists the tags, capabilities, commands, batches and metadata keys that
 {% for type in site.data.registry %}
 <a href="#{{ type.name | slugify }}">{{ type.name }}</a>
 {% endfor %}
+{% for type in site.data.metadata_registry %}
```

Comment:
i might've read over it or it's already somewhere in the code, but a link to the spec(s) would probably be useful for reference

Reply:
there are no specs to link to right now, but when/if we do add them in the future I'll definitely add a column for 'em

Reply:
Well, you're linking to PRs in the fail codes section.

Reply:
When there's a PR that lays out in-detail how particular keys should act and be used, will definitely make sense to link it here.